script/manifest/manifestmodifier.js

```diff
@@ -61,23 +72,36 @@ define('bigscreenplayer/manifest/manifestmodifier',
         };
       }
 
-      if (baseUrl.match(/^https?:\/\//)) {
+      function removeUnusedPeriodAttributes () {
+        if (manifest.Period && manifest.Period.BaseURL) delete manifest.Period.BaseURL;
+        if (manifest.Period && manifest.Period.BaseURL_asArray) delete manifest.Period.BaseURL_asArray;
```

Comment:
Will this mean that in the case that a manifest _did_ contain a `BaseURL_asArray` attribute, we'd always delete it?

Reply:
It always gets removed from the `manifest.Period` but is always set on `manifest.BaseURL_asArray`.

Reply:
What happens if a manifest already has both a `manifest.Period` and `manifest.BaseURL_asArray`? Would we want to be deleting/overriding those?

Reply:
Picked up offline!